posts/2017-11-01-serverless-v1.24.0.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Serverless v1.24 - Alexa Smart Home event source, Print content of resolved serverless config file
+description: Alexa Smart Home event source, update checker, enhancements, bug fixes and more added in the Serverless Framework v1.24 release.
+date: 2017-11-01
+layout: Post
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework_v1.24.jpg
+authors:
+  - PhilippMuns
+---
+
+<img align="right" src="https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/framework_v1.24.jpg" width="210px" >
+
+Today we're proud to announce Serverless Framework v1.24.
+
+v1.24 is a special release—it's the first one which was entirely run by our recently-onboarded Serverless Framework v1 maintainers!
```

Comment:
Maybe a bit more clarity on maintainers:

v1.24 is a special release.  We recently made a few of the Serverless Framework's major open-source contributors official maintainers and this is the first release which was run entirely by them.